### PR TITLE
UTOPIA-479: UI table design

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Implements a data table UI for the PIA List
- Adds `bcgovbtn__tertiary` button css class, as per the Figma files
 
 ## Note
 This PR contains a __dummy data__ array and a __navlink__ for ease of access for testing. Upon final PR approval, __I will remove both of these.__
 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Visually assessed accuracy against Figma design.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [x] New and existing unit tests pass locally with my changes
- [x] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by binggaofw** [comment]
your implementation is a little different compared with Figma. 

 Figma
Screen Shot 2022-12-06 at 7 29 57 PM
Current solution
Screen Shot 2022-12-06 at 7 30 06 PM

**Comment by Bottle7** [comment]
> your implementation is a little different compared with Figma.
> 
> Figma Screen Shot 2022-12-06 at 7 29 57 PM Current solution Screen Shot 2022-12-06 at 7 30 06 PM

Please be specific. Some things appear different due to how the design frames are scaled. For instance, the 'List of PIAs' heading is actually the same font-size in both but appears different due to scaling.

**Comment by binggaofw** [comment]
the nav bar wording, the design is List of PIA, your code is PIA list. also the design display the "List of PIAs" on left side of NavBar.

**Comment by Bottle7** [comment]
I see. So the issue was with the navbar and not the actual table/list. Either way, I have rectified this because it will need to be this way for all of the other UI tickets as well.

**Review by binggaofw — APPROVED** [review]

File: src/frontend/src/sass/_table.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    & > * > tr > * {
+      padding-block: 1.2em;
+      padding-inline: 1.5em;
+    }
+
+    & > thead > tr {
+      border-bottom: 2px solid $accent-colour;
+    }
+
+    & > tbody > tr > td:nth-child(4) {
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_table.scss:16]: I don't think `nth-child(4)` is intuitive enough to locate the status. Can we please use the className instead?

File: src/frontend/src/sass/_table.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        height: 0.5em;
+        border-radius: 50%;
+        background-color: green;
+        align-self: center;
+        display: inline-block;
+        margin-top: 0.17em;
+      }
+    }
+
+    & > tbody > tr > td:nth-child(5) {
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_table.scss:35]: same for this ^^

**Pull request merged**